Makefile

```diff
@@ -438,7 +437,7 @@ define TARGET_RULE
 	    @echo 'Warning: Deprecated target name $(1) specified'
 	    @echo
 	    @echo 'Please use $(1).[$(subst $(space),|,$(MXE_LIB_TYPES))] instead'
-	    @echo 'See index.html for further information'
+	    @echo 'See LICENSE.md for licensing information.'
```

Comment:
I believe the automatic translation went slightly too far here.

If somebody tried to run a deprecated target, we would send them to the actual documentation (docs/index.html), not the licensing page, wouldn't we?